src/components/NewUserDashbord/index.js

```diff
@@ -44,22 +44,13 @@ const NewUserDashbord = () => {
     });
   }, []);
 
-  const pathwayData = [];
-  data &&
-    data.pathways &&
-    data.pathways.forEach((pathway) => {
-      if (pathway.code !== "PRCRSE" || pathway.path) {
-        const obj = {
-          id: pathway.id || null,
-          title: pathway.name || pathway.title,
-          description: pathway.description,
-          image: pathway.image || pathway.logo,
-          link: pathway.path || null,
-          type: "internal",
-        };
-        pathwayData.push(obj);
-      }
-    });
+  const MiscellaneousCourses = data?.pathways.filter(
+    (item) => item.name === "Miscellaneous Courses"
+  );
+
+  const pathwayData = data?.pathways
+    .filter((item) => item.name !== "Miscellaneous Courses")
+    .concat(MiscellaneousCourses);
```

Comment:
Is the reason for this code to include the Miscellaneous courses pathway at the end? We'll probably want to avoid explicit mention of the name in case we decide to change its name or offer translations into other languages. Perhaps we could `import` and use `PATHWAY_INFO` to determine whether it should be at the end of not. Also, we should probably rename the variable to indicate that it's a pathway name and not the collection of miscellaneous courses such as Web Scraping 101.

For example:
```suggestion
  const miscellaneousPathway = data?.pathways.filter(
    (pathway) => PATHWAY_INFO.some((miscPathway) => pathway.name === miscPathway.name)
  );

  const pathwayData = data?.pathways
    .filter((pathway) => !miscellaneousPathway.includes(pathway))
    .concat(miscellaneousPathway);
```

Reply:
Okay, I will change it but I will ask the backend to send this course from the backend only.